bgpd/bgp_main.c

```diff
@@ -75,6 +75,7 @@ static const struct option longopts[] = {
 	{"ecmp", required_argument, NULL, 'e'},
 	{"int_num", required_argument, NULL, 'I'},
 	{"no_zebra", no_argument, NULL, 'Z'},
+	{"socket_size", required_argument, NULL, 's'},
```

Comment:
wouldn't it be more understandable having `sndbuf` here? and maybe `rcvbuf` (in the future).

Reply:
I see no point in not having it symmetric.  do you expect a need for it not to be?

Reply:
In most cases I think that's fine, I'm just trying to find some cases with different sizes. There are cases when the kernel doesn't ACK or keepalives messages unless the buffer is empty or so. It's just a minor note from me, nothing special :+1:

Reply:
Only if you change sndbuf then rcvbuff leaves as default anyway :) Hence, they are asymmetric.

Reply:
fixed